README.md

```diff
@@ -113,244 +112,128 @@ bash test/test_cleanup.sh
 Now that your virtual environment is set up and the workflow is deployed and
 tested, you can go ahead and run the workflow on your samples.
 
-But first, here is a brief description of what the workflow does:
-
-### Workflow description
-
-The workflow consist on one main `Snakefile` 
-gathering three different subworkflows,
-**_PREPARE_**, **_MAP_** and **_QUANTIFY_**. 
-Each one of them is implemented in an individual Snakemake file
-and fully containerized.    
-
-
-#### _PREPARE_
-
-The first subworkflow, **_PREPARE_**, downloads and processes "genome resources"
-from the publicly available repositories [Ensembl][ensembl] and
-[miRBase][mirbase] according to your instructions. Resources are then processed
-to prepare indexes and other contingent resources that will be used in later
-steps.
-
-#### _MAP_
-
-The second subworkflow, **_MAP_**, aligns the user-provided short read smallRNA-seq
-libraries against the references generated with the **_PREPARE_** subworkflow. For
-increased fidelity it uses two separate aligning tools, [Segemehl][segemehl]
-and our in-house tool [Oligomap][oligomap]. In both cases, reads are aligned
-separately to the genome and the transcriptome. Afterwards, alignments are
-merged in a way that only the best alignment (or alignments) of each read are
-kept.
-
-#### _QUANTIFY_
-
-The third and final subworkflow, **_QUANTIFY_**, quantifies miRNA expression by
-intersecting the alignments from the **_MAP_** subworkflow with the annotations
-generated in the **_PREPARE_** subworkflow. Intersections are computed with
-[`bedtools`][bedtools] for one or multiple of mature, primary transcripts and
-isomiRs. Reads consistent with each miRNA are counted and tabulated.
-
-
-The schema below is a visual representation of
-an example run of the whole workflow:
-
-> ![rule-graph][rule-graph]
-
-
-### Running the workflow
-
 Assuming that you are currently inside the repository's root directory, 
-create a directory from which you will run your workflow and
-name it whatever you want e.g., `MY_ANALYSIS` and 
-head to it.
+create a directory from which you will run your workflow and name it whatever 
```

Comment:
They don't need to do that. They can totally let their files stay where they are as long as Snakemake can access them.

Reply:
Could it be more a suggestion than a mandatory step?

Reply:
You could add a quote comment where you suggest that all inputs for a given run (or hard links pointing to them) are kept inside a dedicated directory, for example under the MIRFLOWZ root directory. And you can say that this makes it easer to keep the data together, reproduce an analysis and set up Singularity access to them. But do this only once at the beginning of the "Preparing inputs" section.

Reply:
I don't understand what do you mean by "set up Singularity access to them"...

Reply:
Singularity can only access data from the file system if you explicitly mount a directory containing them via the `--bind my/directory` (relative or absolute). For example, if you mount the current working directory, Singularity can access all files in that directory, including in its subdirectories. Of course, in principle, you could just mount the file system's root directory with `/`, and Singularity would be able to access the entire file system. However, for security reasons it is strongly recommended to only mount the lowest level directory or directories necessary to access all input files.

Therefore, if your workflow requires multiple inputs and they are all spread across different paths of the file system, then you'd either need to mount multiple directories, or mount a single directory that is the parent of all the subdirectories containing the input files, but which may be on a higher level then necessary.

For example, assume your file system looks like this, you are currently in `/home/my_home/mirflowz` and the workflow requires access to files `input_?`:

```
/
    home/
        my_home/
            Documents/
                input_A
                Private/
                    # all kinds of private docs
            Downloads/
                input_B
            Secret/
                # all kinds of secret docs
            Work/
                mirflowz/
                    input_C
```

In this case you'd need to either mount `.` (may be mounted by default, not sure), `../../Documents` _and_ `../../Downloads`, or mount `../..` (or `/home/my_home` when using absolute paths).

Two issues with that:

1. In the first case your `--bind` command becomes more complicated, because you need to provide multiple arguments.
2. In both cases you'd give the container access to more data than necessary, e.g., the data under `/home/my_home/Documents/Private` (in both cases) and `/home/my_home/Secret` (in the second case).

If, however, all the inputs are under the current working directory, then you'd only need to mount `.` (or `/home/my_home/Work/mirflowz` when using absolute paths).

Does that make more sense now? Of course you don't need to describe this in detail. Just a small sentence, or half sentence, maybe with a link to the Singularity docs, will be enough)

P.S.: Check the test script to see how to pass the `--bind` parameter to Singularity via Snakemake.